manifests/istio-operator-template.yaml

```diff
@@ -118,8 +118,8 @@ spec:
             Any modifications are discarded and the resource is reverted to the original state.
         resources:
           limits:
-            cpu: 500m
-            memory: 1024Mi
+            cpu: 4000m
```

Comment:
Why did we change this? Was this committed accidentally?

Reply:
I believe those resources where changed since the time we created this template: https://github.com/kyma-project/kyma/commit/45a4d573e83914fd9d593ea55ead630d648b5c22